README.md

```diff
@@ -34,6 +34,8 @@ If you ever run into a case, where you add a gem or update the version of a gem
 You can use the command `tapioca require` to auto-populate the `sorbet/tapioca/require.rb` file with all the requires found
 in your application. Once the file generated, you should review it, remove all unnecessary requires and commit it.
 
+`tapioca` supports stronger/stricter type generation if class extends `StrongTypeGenerataion` module. `StrongTypeGenerataion` module does not allow the object to be initialize with a bad state by checking each non_nilable attribute to ensure they are not nil. `tapioca`  ActiveRecordColumns RBI generator allows for more strict type signature if the object class extend `StrongTypeGenerataion` by generating sig with exact defined type instead of T.untyped.
```

Comment:
`StrongTypeGeneration` has a typo in all 3 usages.

Reply:
I would prefer this to be lower if it's going to be in the README as it's a tinier detail compared to the main information at this part of the document. If a wiki page won't be visible enough it'd make sense to insert it towards the end such as after `Flags` section?

Reply:
like your suggestion, thanks Kaan!  should I add a header too? what should I use for header? is `StrongTypeGeneration Support` ok?

Reply:
Yes we'd need a header as it doesn't really fit any headers right now. 
`Strong typing option for ActiveRecord column methods` is informative enough I think.